common/djangoapps/student/views/dashboard.py

```diff
@@ -585,25 +585,35 @@ def student_dashboard(request):
     ecommerce_service = EcommerceService()
     inverted_programs = meter.invert_programs()
 
-    urls, program_data = {}, {}
+    urls, programs_data = {}, {}
     bundles_on_dashboard_flag = WaffleFlag(WaffleFlagNamespace(name=u'student.experiments'), u'bundles_on_dashboard')
 
     if bundles_on_dashboard_flag.is_enabled():
-        programs_data = meter.programs
-
-        if programs_data and inverted_programs and inverted_programs.values():
-            program_uuid = inverted_programs.values()[0][0]['uuid']
-            meter.programs = [get_programs(request.site, uuid=program_uuid)]
-            program_data = meter.programs[0]
-            program_data = ProgramDataExtender(program_data, request.user).extend()
-
-            skus = program_data.get('skus')
-
-            urls = {
-                'commerce_api_url': reverse('commerce_api:v0:baskets:create'),
-                'buy_button_url': ecommerce_service.get_checkout_page_url(*skus)
-            }
-            urls['completeProgramURL'] = urls['buy_button_url'] + '&bundle=' + program_data.get('uuid')
+        programs_data = {}
+        if inverted_programs and inverted_programs.items():
+            for program in inverted_programs.values():
+                try:
+                    program_uuid = program[0]['uuid']
+                    program_data = get_programs(request.site, uuid=program_uuid)
```

Comment:
i disagree, this doesn't return a class that extends program data, it returns the same data with a few keys added. i think it makes to rename variables when it is something semantically different, rather than a different value of the same thing

for instance, in the code i had before, i don't think it would make sense to do something like
price_string = ''
longer_price_string = price_string + 'stuff'
even_longer_price_string = longer_price_string + 'stuff'

however...i don't care that much so if you still want me to i'll rename